### PR TITLE
Chore: Update outdated npm packages and fix issues from audit

### PR DESCRIPTION
**Pull request by chosww**
* [x] This pull request has been tested by running `npm run test` without errors
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

Update outdate npm packages and fix issues from the audit.

### CODE REVIEW

**Comment by cindyli** [comment]
@chosww running `npm outdated` in this branch still reports a bunch of outdated packages. This branch requires not only updating `package-lock.json` but also `package.json`.

After updating outdated packages in `package.json` to the latest version, you probably have known but just in case, if you delete `package-lock.json` then run `npm install`, `package-lock.json` will be recreated with all latest dependencies.

**Comment by cindyli** [comment]
> what should I do about "stylelint-config-fluid"? right now it is set to specific version of config-fluid. "8b6f7964ceb21acf690efb07b28fdda589ecfdd2" and where do I get the hash for the latest version?

Find the latest version from [npmjs.com](https://www.npmjs.com/package/stylelint-config-fluid). You can also look into [its github repo](https://github.com/fluid-project/stylelint-config-fluid) for commits.

**Comment by chosww** [comment]
Hey @cindyli, I am having issue with peer dependency issue between eleventy and eleventy-plugin-error-overlay. Prior to this PR, we were using eleventy 2.0.0, and with the update, it is updated to 2.0.1; however, eleventy-plugin-error-overlay requires eleventy^1.0.0, which both 2.0.0 and 2.0.1 should work but npm install throws an error. When I specify eleventy to be ^1.0.0, npm install doesn't throw the error, but now it has vulnerability issue. It's weird that I am seeing the peer dependency issue between eleventy-plugin-error-overlay and eleventy because it wasn't complaining before. Could you provide some guidance on how should I approach this issue please?

**Comment by cindyli** [comment]
> Hey @cindyli, I am having issue with peer dependency issue between eleventy and eleventy-plugin-error-overlay.

The workaround is to update eleventy to 2.0.1 then run `npm install --legacy-peer-deps` so eleventy-plugin-error-overlay ignores the mismatched dependency. But this brings a risk of breaking the build. Can you give a try? In the meanwhile, please file an issue with [eleventy-plugin-error-overlay](https://github.com/stevenpetryk/eleventy-plugin-error-overlay). Hopefully this plugin can be updated to match the eleventy upgrade. Thanks.

**Comment by chosww** [comment]
> > Hey @cindyli, I am having issue with peer dependency issue between eleventy and eleventy-plugin-error-overlay.
> 
> The workaround is to update eleventy to 2.0.1 then run `npm install --legacy-peer-deps` so eleventy-plugin-error-overlay ignores the mismatched dependency. But this brings a risk of breaking the build. Can you give a try? In the meanwhile, please file an issue with [eleventy-plugin-error-overlay](https://github.com/stevenpetryk/eleventy-plugin-error-overlay). Hopefully this plugin can be updated to match the eleventy upgrade. Thanks.

Thanks @cindyli, running npm install with legacy-peer-deps works locally, but CI is still failing because we are not running npm install with the flag. Also, I opened an issue to eleventy-plugin-error-overlay about this issue: https://github.com/stevenpetryk/eleventy-plugin-error-overlay/issues/5

**Comment by greatislander** [comment]
@cindyli and @chosww — it looks like you can run `npm ci --legacy-peer-deps` and also specify it in the `.npmrc` file:

> If you're using `--legacy-peer-deps` with `npm install`, make sure you also use it with `npm ci`, perhaps by setting it in your `.npmrc`.

File: package.json [hunk 1]
```diff
@@ -65,34 +65,34 @@
...
-    "sass": "1.58.3",
-    "sass-loader": "13.2.0",
+    "rimraf": "5.0.1",
+    "sass": "1.62.1",
+    "sass-loader": "13.3.1",
     "start-server-and-test": "2.0.0",
-    "stylelint": "14.16.1",
+    "stylelint": "15.6.2",
     "stylelint-config-fluid": "github:fluid-project/stylelint-config-fluid#8b6f7964ceb21acf690efb07b28fdda589ecfdd2",
-    "stylelint-config-standard-scss": "6.1.0"
```
Comment (greatislander) [inline package.json:96]: Adjusting these dependencies resolves the remaining CI error.

```suggestion
    "stylelint": "14.16.1",
    "stylelint-config-fluid": "1.0.0",
    "stylelint-config-standard-scss": "6.1.0"
```

Comment (chosww) [reply package.json]: Hey @greatislander, after these adjustments, I am still getting js lint error from infusion-uio.js. Is this happening to you as well?

Comment (chosww) [reply package.json]: Oh, but the checks on CI has passed, I am seeing the issues on local env.

**Review by greatislander — APPROVED** [review]

**Comment by greatislander** [comment]
@chosww can you update eleventy-plugin-fluid to 1.0.1 and eleventy to 2.0.1 now that the bug fix is done?

File: package.json (hunk 1 above)
Comment (greatislander) [reply package.json]: @chosww Can you reset your local environment? Unstage any local changes, run `npm ci` and try to lint again?

**Comment by chosww** [comment]
Hi @g

> @chosww can you update eleventy-plugin-fluid to 1.0.1 and eleventy to 2.0.1 now that the bug fix is done?

For sure, thanks for working on the update :)

**Review by greatislander — APPROVED** [review]

**Comment by cindyli** [comment]
Running `npm outdated` in this branch still shows some outdated packages. Shall those be upgraded too?

```
backstopjs                        6.2.1    6.2.1   6.2.2  node_modules/backstopjs                      idrc
concurrently                      8.1.0    8.1.0   8.2.0  node_modules/concurrently                    idrc
eslint                           8.41.0   8.41.0  8.42.0  node_modules/eslint                          idrc
move-file                         2.1.0    2.1.0   3.1.0  node_modules/move-file                       idrc
sass                             1.62.1   1.62.1  1.63.4  node_modules/sass                            idrc
sass-loader                      13.3.1   13.3.1  13.3.2  node_modules/sass-loader                     idrc
stylelint                       14.16.1  14.16.1  15.7.0  node_modules/stylelint                       idrc
stylelint-config-standard-scss    6.1.0    6.1.0   9.0.0  node_modules/stylelint-config-standard-scss  idrc
```

**Comment by greatislander** [comment]
@cindyli The `move-file` update is not compatible (it switched to ES Modules) and the `stylelint` updates introduce a bunch of new/modified rules which would necessitate many SCSS changes. The other updates can be applied safely, though.

File: package.json [hunk 2]
```diff
@@ -53,46 +53,46 @@
...
+    "lint-staged": "13.2.2",
     "move-file": "2.1.0",
     "netlify-cms-proxy-server": "1.3.24",
     "resolve-url-loader": "5.0.0",
-    "rimraf": "4.3.1",
-    "sass": "1.58.3",
-    "sass-loader": "13.2.0",
+    "rimraf": "5.0.1",
+    "sass": "1.62.1",
+    "sass-loader": "13.3.1",
```
Comment (greatislander) [inline package.json:92]:
```suggestion
    "sass": "1.63.4",
    "sass-loader": "13.3.2",
```

File: package.json [hunk 3]
```diff
@@ -53,46 +53,46 @@
...
-    "@sindresorhus/slugify": "2.2.0",
-    "backstopjs": "6.1.4",
+    "@commitlint/cli": "17.6.5",
+    "@commitlint/config-conventional": "17.6.5",
+    "@sindresorhus/slugify": "2.2.1",
+    "backstopjs": "6.2.1",
     "cross-env": "7.0.3",
     "eleventy-plugin-error-overlay": "1.0.0",
-    "eslint": "8.35.0",
+    "eslint": "8.41.0",
```
Comment (greatislander) [inline package.json:79]:
```suggestion
    "eslint": "8.42.0",
```

File: package.json [hunk 4]
```diff
@@ -53,46 +53,46 @@
...
   },
   "devDependencies": {
-    "@commitlint/cli": "17.4.4",
-    "@commitlint/config-conventional": "17.4.4",
-    "@sindresorhus/slugify": "2.2.0",
-    "backstopjs": "6.1.4",
+    "@commitlint/cli": "17.6.5",
+    "@commitlint/config-conventional": "17.6.5",
+    "@sindresorhus/slugify": "2.2.1",
+    "backstopjs": "6.2.1",
```
Comment (greatislander) [inline package.json:76]:
```suggestion
    "backstopjs": "6.2.2",
```

File: package.json [hunk 5]
```diff
@@ -53,46 +53,46 @@
     ]
   },
   "dependencies": {
-    "@11ty/eleventy": "2.0.0",
+    "@11ty/eleventy": "2.0.1",
     "@11ty/eleventy-navigation": "0.3.5",
     "@11ty/eleventy-plugin-rss": "1.2.0",
-    "concurrently": "7.6.0",
+    "concurrently": "8.1.0",
```
Comment (greatislander) [inline package.json:59]:
```suggestion
    "concurrently": "8.2.0",
```

**Review by greatislander — CHANGES_REQUESTED** [review]

**Review by cindyli — APPROVED** [review]

**Pull request merged**